Загрузка

skan7575 opened the pull request.

Commit:
Загрузка

Commit:
Первая проверка верстки

coolswood commented:
# Резюме по работе:
## Верстка и JSX

### Критерии, влияющие на работоспособность
- [x] <!--12--> Все блоки и элементы из макета сверстаны. Размеры и расположение не отличаются от макета на более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета.
`Цвет фона всплывающего меню не соответствует макету`
`Отсутствует состояние кнопки в карточке "активированное»`
`Отсутствует состояние кнопки в карточке "крестик»`
`Отсутствует кнопка "еще" на стр /movies`

`Нет такого состояния расположения блоков`
<img width="350" alt="Screenshot 2022-11-07 at 17 08 07" src="https://user-images.githubusercontent.com/25247802/200344644-054f4c57-4267-4d0a-a0d7-544e51aeed6d.png">

- [x] <!--10.55--> Проект адаптирован под различные разрешения экрана. Горизонтальный скролл не возникает на разрешениях от 320 пикселей и больше. Нельзя скрывать полосу прокрутки свойством overflow: hidden.
- [x] <!--9.55--> Корректно работает навигация между страницами и ссылки на внешние ресурсы: ни одна ссылка не ведёт в пустоту или на якорь, внешние ссылки открываются в новой вкладке. На все страницы проекта можно попасть, а скрытые блоки можно отобразить.
`ссылки class="portfolio__list-item portfolio__list-item_underline"  ведут в пустоту (404) Проверьте, открыт ли аккаунт`

- [x] <!--5.55--> Именование классов сделаны по БЭМ.
- [x] <!--8.55--> Отзывчивая вёрстка, которая корректно тянется на всех промежуточных разрешениях.
`Контент страницы прилипает к краям страницы (проверьте весь проект)`
`Иконка стрелка обрезается`
<img width="358" alt="Screenshot 2022-11-07 at 17 09 29" src="https://user-images.githubusercontent.com/25247802/200344713-af6942f1-bb92-4718-85b7-1101f2ba98e6.png">


- [x] <!--3.38--> В коде используется семантическая разметка: применяются семантические теги, выбор элементов при вёрстке корректен (параграф должен быть параграфом, список — списком); структура DOM-дерева состоит не только из контейнеров div.
`Весь контент между header и footer нужно заключить в тег main, как в тег для уникального контента страницы.`
`Всем кнопкам всегда нужно указывать тип. Потому что по-умолчанию это submit и такая кнопка при клике может отправить форму.`

- [x] <!--3.38--> Для позиционирования элементов выбран верный подход, описанный корректным синтаксисом.
- [x] <!--3.38--> Каркас макета реализован на Flex layout или Grid layout;x
- [x] <!--3.38--> Инфраструктурные файлы проекта созданы через CRA.
- [x] <!--3.38--> Работа модальных окон настроена.
- [x] Разметка портирована в JSX:
	- [x] <!--3.38--> разметка заключена в ( ), лишние пустые <div> для этого не используются;
	- [x] <!--3.38--> разметка вынесена в соответствующие ей компоненты.
- [x] В проекте есть:
	- [x] <!--3.38--> директория images/ с картинками,
	- [x] <!--3.38--> директория components/ с JS и CSS-файлами компонентов,
	- [x] <!--3.38--> директория fonts/ со шрифтами.

### Хорошие практики
- [x] <!--2.14--> Кнопки, инпуты и ссылки реализованы во всех состояниях, указанных в макете. 
- [x] <!--2.14--> В макете есть одинаковые элементы. Для их оформления должен быть переиспользован один компонент.
- [x] <!--2.14--> Шрифты подключены через @font-face.
- [x] <!--2.14--> Иконки экспортированы в формате SVG.
- [x] <!--2.14--> Элементы формы должны выделяться, когда на них установлен фокус. Состояние фокуса, а также outline стилизованы в соответствии с макетом или на своё усмотрение.
- [x] <!--2.14--> У формы должны быть заданы плейсхолдеры, валидированы обязательные поля.
`У формы поиска фильмов нет свойства required`
- [x] <!--2.14--> reset.css запрещен.

### Рекомендации
- [x] <!--1.68--> Для каждого шрифта указаны альтернативные варианты из системных шрифтов.
- [x] <!--1.67--> Для изображений задан атрибут alt с подходящий значением.
`Каждый тег img должен содержать заполненный атрибут alt`
- [x] <!--1.67--> Растровые и векторные изображения оптимизированы;

## Количество баллов: 63

## Доп. Комментарии:

Commit:
1. изменен цвет фона всплывающего меню
2. Сделаны кнопки сохранение и удаления карточек из избранного
3. Добавлена кнопка еще на стр поиска фильмов
4. Исправлен главный блок промо для мобильных устройств
5. Исрпавлены ссылки
6. Исправлена стрелочка
7. Исрпавлена семантика верстки, проставленны типы для кнопок также main для каждой страницы
8. Добавлен обазательное поле для поиска фильмов
9, Проставлены alt to img

Commit:
1. изменен цвет фона всплывающего меню
2. Сделаны кнопки сохранение и удаления карточек из избранного
3. Добавлена кнопка еще на стр поиска фильмов
4. Исправлен главный блок промо для мобильных устройств
5. Исрпавлены ссылки
6. Исправлена стрелочка
7. Исрпавлена семантика верстки, проставленны типы для кнопок также main для каждой страницы
8. Добавлен обазательное поле для поиска фильмов
9, Проставлены alt to img

coolswood commented:
# Резюме по работе:
## Верстка и JSX

### Критерии, влияющие на работоспособность
- [x] <!--12--> Все блоки и элементы из макета сверстаны. Размеры и расположение не отличаются от макета на более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета.
- [x] <!--10.55--> Проект адаптирован под различные разрешения экрана. Горизонтальный скролл не возникает на разрешениях от 320 пикселей и больше. Нельзя скрывать полосу прокрутки свойством overflow: hidden.
- [x] <!--9.55--> Корректно работает навигация между страницами и ссылки на внешние ресурсы: ни одна ссылка не ведёт в пустоту или на якорь, внешние ссылки открываются в новой вкладке. На все страницы проекта можно попасть, а скрытые блоки можно отобразить.
- [x] <!--5.55--> Именование классов сделаны по БЭМ.
- [ ] <!--8.55--> Отзывчивая вёрстка, которая корректно тянется на всех промежуточных разрешениях.
`Контент страницы прилипает к краям страницы (проверьте весь проект)`
<img width="239" alt="Screenshot 2022-11-09 at 20 48 54" src="https://user-images.githubusercontent.com/25247802/200915710-452d7389-8709-4fa7-9f48-b064f4d528a7.png">



- [ ] <!--3.38--> В коде используется семантическая разметка: применяются семантические теги, выбор элементов при вёрстке корректен (параграф должен быть параграфом, список — списком); структура DOM-дерева состоит не только из контейнеров div.
`Весь контент между header и footer нужно заключить в тег main, как в тег для уникального контента страницы.`

- [x] <!--3.38--> Для позиционирования элементов выбран верный подход, описанный корректным синтаксисом.
- [x] <!--3.38--> Каркас макета реализован на Flex layout или Grid layout;x
- [x] <!--3.38--> Инфраструктурные файлы проекта созданы через CRA.
- [x] <!--3.38--> Работа модальных окон настроена.
- [x] Разметка портирована в JSX:
	- [x] <!--3.38--> разметка заключена в ( ), лишние пустые <div> для этого не используются;
	- [x] <!--3.38--> разметка вынесена в соответствующие ей компоненты.
- [x] В проекте есть:
	- [x] <!--3.38--> директория images/ с картинками,
	- [x] <!--3.38--> директория components/ с JS и CSS-файлами компонентов,
	- [x] <!--3.38--> директория fonts/ со шрифтами.

### Хорошие практики
- [x] <!--2.14--> Кнопки, инпуты и ссылки реализованы во всех состояниях, указанных в макете. 
- [x] <!--2.14--> В макете есть одинаковые элементы. Для их оформления должен быть переиспользован один компонент.
- [x] <!--2.14--> Шрифты подключены через @font-face.
- [x] <!--2.14--> Иконки экспортированы в формате SVG.
- [x] <!--2.14--> Элементы формы должны выделяться, когда на них установлен фокус. Состояние фокуса, а также outline стилизованы в соответствии с макетом или на своё усмотрение.
- [x] <!--2.14--> У формы должны быть заданы плейсхолдеры, валидированы обязательные поля.
- [x] <!--2.14--> reset.css запрещен.

### Рекомендации
- [x] <!--1.68--> Для каждого шрифта указаны альтернативные варианты из системных шрифтов.
- [x] <!--1.67--> Для изображений задан атрибут alt с подходящий значением.
- [x] <!--1.67--> Растровые и векторные изображения оптимизированы;

## Количество баллов: 88

## Доп. Комментарии:
